Dockerfile

```diff
@@ -17,12 +17,30 @@ RUN mkdir /opt/android-ndk-tmp
 RUN cd /opt/android-ndk-tmp && wget -q http://dl.google.com/android/repository/android-ndk-r12-linux-x86_64.zip
 # uncompress
 RUN cd /opt/android-ndk-tmp && unzip -q android-ndk-r12-linux-x86_64.zip
-# move to it's final location
+# move to its final location
 RUN cd /opt/android-ndk-tmp && mv ./android-ndk-r12 /opt/android-ndk
 # remove temp dir
 RUN rm -rf /opt/android-ndk-tmp
 # add to PATH
 ENV PATH ${PATH}:${ANDROID_NDK_HOME}
+
+
+# ------------------------------------------------------
+# --- Android CMake
+
+# download
+RUN mkdir /opt/android-cmake-tmp
+RUN cd /opt/android-cmake-tmp && wget -q https://dl.google.com/android/repository/cmake-3.6.3155560-linux-x86_64.zip -O android-cmake.zip
+# uncompress
+RUN cd /opt/android-cmake-tmp && unzip -q android-cmake.zip -d android-cmake
+# move to its final location
+RUN cd /opt/android-cmake-tmp && mv ./android-cmake ${ANDROID_HOME}/cmake
```

Comment:
hum, true, I tried without the version number just because of the error message they showed me when I tried to build without CMake:
```
Expected CMake executable at /Users/xx/Library/Android/sdk/cmake/bin/cmake.
```
It works outside the version folder, I need to recheck to see if it builds using the Android Studio folder config, probable it will